bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/wizard/NewIDFProjectWizard.java

```diff
@@ -165,12 +168,13 @@ private void createDefaultDebugConfig()
 
 		PageChangingEvent pageChangingEvent = new PageChangingEvent(wizard, wizard.getStartingPage(), editPage);
 		editPage.handlePageChanging(pageChangingEvent);
-
 		wizard.performFinish();
 
 		try
 		{
-			wizard.getWorkingCopy().doSave();
+			String originalName = wizard.getWorkingCopy().getName();
+			String debugConfigName = originalName.substring(0, originalName.lastIndexOf("Configuration")) + " Debug"; //$NON-NLS-1$ //$NON-NLS-2$
+			wizard.getWorkingCopy().copy(debugConfigName).doSave();
```

Comment:
After rename, you might need to refresh the project to avoid sync issues with file system project state.

Here is the case:

1. Create a project
2. Delete a project 
3. Choose to delete all configurations in the popup window
4. Still you will see "project Debug" configurations in the configuration drop-down

Reply:
do not see this issue during test.